client/components/count/index.jsx

```diff
@@ -1,36 +1,3 @@
-import classnames from 'classnames';
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
-import formatNumberCompact from 'calypso/lib/format-number-compact';
+import { Count } from '@automattic/components';
```

Comment:
**Note:**

I haven't implemented this yet, but if Count is false-y, render the original count component